client/data/settings/actions.js

```diff
@@ -97,6 +97,18 @@ export function updateAccountStatementDescriptor( accountStatementDescriptor ) {
 	} );
 }
 
+export function updateIsShortStatementEnabled( isEnabled ) {
```

Comment:
Nit: I'd vote for maintaining the full term "StatementDescriptor", even though the function name becomes longer, just to be 100% clear, because the word "statement" might not refer to the bank statement or the statement descriptor, when you read everything.

If you agree, that could apply also for `useIsShortStatementEnabled` and `getIsShortStatementEnabled`.

Reply:
Agreed, I've changed them in 04bcfa8.